cmd/metrics-resource.go

```diff
@@ -280,82 +281,70 @@ func collectDriveMetrics(m madmin.RealtimeMetrics) {
 func collectLocalResourceMetrics() {
 	var types madmin.MetricType = madmin.MetricsDisk | madmin.MetricNet | madmin.MetricsMem | madmin.MetricsCPU
 
```

Comment:
https://github.com/minio/minio/pull/19762 PR sent here, its still part of this to allow testing.